examples/text-generation/README.md

```diff
@@ -73,6 +74,7 @@ with
 - `number_of_iterations` the number of iterations to perform in the benchmark
 - `use_hpu_graphs` enables HPU graphs which are recommended for faster latencies
 - `use_kv_cache` enables a key-value cache to speed up the generation process.
+- `enable_sampling` enables sampling algorithm for the generation process.
```

Comment:
```suggestion
- `do_sample` enables sampling algorithm for the generation process.
```
The name of the argument is `do_sample`.

Reply:
done